src/control/cmd/daos_agent/security_rpc.go

```diff
@@ -59,17 +59,18 @@ func (m *SecurityModule) getCredential(session *drpc.Session) ([]byte, error) {
 
 	signingKey, err := m.config.PrivateKey()
 	if err != nil {
-		m.log.Error(err.Error())
+		m.log.Errorf("%s: failed to get signing key: %s", info, err)
```

Comment:
I'm not sure how uid/gid is sensitive info. My thought was that including it could help an admin with tracking down a problem with a particular user's application, etc. Worth noting that this is not a raw/debug representation of the DomainInfo struct; I added a [new stringer](https://github.com/daos-stack/daos/pull/12017/files#diff-af690e912abd119c3eef7575b9d02915494ff41e2782990571582747444fd04dR44) that doesn't include the credential. If you think this needs more consideration, though, I'm happy to pull it out into a separate PR for extra design review.

Reply:
I'm okay with leaving it as-is, then.